crates/typst/src/model/reference.rs

```diff
@@ -177,8 +177,14 @@ impl Show for Packed<RefElem> {
 
         let elem = elem.at(span)?;
 
-        if elem.func() == FootnoteElem::elem() {
-            return Ok(FootnoteElem::with_label(target).pack().spanned(span));
+        if elem.is::<FootnoteElem>() {
```

Comment:
Why not use `Content::get_ref` here to avoid the unwrap later on?

Reply:
I cannot find the `Content::get_ref` function. Could you please tell me where it is?

Reply:
I see `Packed::<T>::from_ref`. Guess you mean we can use this?

Reply:
The canonical version is `elem.to_packed`.